extensions/data-workspace/src/dialogs/dialogBase.ts

```diff
@@ -17,7 +17,7 @@ interface Deferred<T> {
 
 export abstract class DialogBase {
 	protected _toDispose: vscode.Disposable[] = [];
-	protected _dialogObject: azdata.window.Dialog;
+	public _dialogObject: azdata.window.Dialog;
```

Comment:
since this is a public property now, do you mind update its name to remove the leading '_'?